ui/dist/index.html

```diff
@@ -6,7 +6,7 @@
 
         <link href='https://fonts.googleapis.com/css?family=Raleway:600,300' rel='stylesheet' type='text/css'>
         <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400italic,600italic,600' rel='stylesheet' type='text/css'>
-        <link href="./css/all.css" rel="stylesheet" type="text/css" media="screen,print" />
+        <link id="theme" href="./themes/light.css" rel="stylesheet" type="text/css" media="screen,print" />
```

Comment:
I mean split the splash screen CSS into it's own file. That way when the app loads, the splash screen and associated CSS can display fine.